nixos/modules/module-list.nix

```diff
@@ -585,7 +585,7 @@
   ./services/networking/dhcpd.nix
   ./services/networking/dnscache.nix
   ./services/networking/dnschain.nix
-  ./services/networking/dnscrypt-proxy.nix
```

Comment:
Add an entry in `nixos/modules/rename.nix` to mark this module as removed

Reply:
Shouldn't that be done via an import in dnscrypt-proxy2.nix?

Reply:
I think it would be more useful in `rename.nix`, because since https://github.com/NixOS/nixpkgs/pull/76857 such `imports` can be removed recursively, meaning if it were defined in `dnscrypt-proxy2.nix` and you'd disable that module, the module deprecation warning would also be disabled, which doesn't make much sense.